mp/src/gameui/OptionsSubVideo.cpp

```diff
@@ -920,31 +860,14 @@ class COptionsSubVideoAdvancedDlg : public vgui::Frame
 				return false;
 			if ( !GameUI().IsInMultiplayer() )
 				return false;
-
-			ConVarRef mat_dxlevel( "mat_dxlevel" );
-			KeyValues *pUserData = m_pDXLevel->GetActiveItemUserData();
-			Assert( pUserData );
-			if ( pUserData && mat_dxlevel.GetInt() != pUserData->GetInt("dxlevel") )
-			{
-				return true;
-			}
-
-			// HDR changed?
-			if ( m_pHDR->IsEnabled() )
-			{
-				ConVarRef mat_hdr_level("mat_hdr_level");
-				Assert( mat_hdr_level.IsValid() );
-				if ( mat_hdr_level.GetInt() != m_pHDR->GetActiveItem() )
-					return true;
-			}
 		}
 		return false;
 	}
 
 private:
 	bool m_bUseChanges;
 	vgui::ComboBox *m_pModelDetail, *m_pTextureDetail, *m_pAntialiasingMode, *m_pFilteringMode;
-	vgui::ComboBox *m_pShadowDetail, *m_pHDR, *m_pWaterDetail, *m_pVSync, *m_pShaderDetail;
+	vgui::ComboBox *m_pShadowDetail, *m_pWaterDetail, *m_pVSync, *m_pShaderDetail;
 	vgui::ComboBox *m_pColorCorrection;
 	vgui::ComboBox *m_pMotionBlur;
 	vgui::ComboBox *m_pDXLevel;
```

Comment:
Why not also get rid of this combo box and the `g_DirectXLevels` array? Since we're hardcoding to 90+, you can safely remove all of the references to the old directX stuff and replace it with a static label (or just remove it entirely, whichever looks cleaner).